modules/shared/components/molecules/CreatePostHeader/CreatePostHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { ICreatePostHeader } from './ICreatePostHeader';
+import Avatar from '../../atoms/Avatar/Avatar';
```

Comment:
I know the team  agreed to name the component `Avatar` but we have a duplication (the word avatar) here `../../atoms/Avatar/Avatar`

when we started the project  we agreed to name it [Avatar/index.tsx] 
and import it from `../../atoms/Avatar`
(In the future if we need to make any change we had to discuss why and what are pros and cons of that)

Why you changed it and what is the argument, benefits. 
(by you I mean all members 😂😂😂 not only @atwfk )